UIKitTemplate/UIKitTemplate/ChooseSizeViewController.swift

```diff
@@ -0,0 +1,112 @@
+// ChooseSizeViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+protocol SizeDelegate: AnyObject {
+    /// передача размера обуви в корзину
+    func sendSize(size: Int)
+}
+
+///  Экран для выбора размера обуви
+final class ChooseSizeViewController: UIViewController {
+    // MARK: - Constants
+
+    enum Constants {
+        static let xmarkImage = "xmark"
+        static let titleText = "Выберете размер"
+        static let nameFont = "Verdana-Bold"
+        static let newItemsLabelSize = 16
+        static let startYButton = 77
+        static let startYView = 103
+        static let stepY = 37
+    }
+
+    private let sizeSelection = [35, 36, 37, 38, 39]
```

Comment:
```suggestion
    private let availableSizes = [35, 36, 37, 38, 39]
```
у массивов должно быть окончание -s